tutor/src/screens/task/exercise-task-header.js

```diff
@@ -95,9 +113,9 @@ class ExercisesTaskHeader extends React.Component {
           </TutorLink>
         </StyledBackLink>
         <StyledHeadingTitle>
-          <span>{ux.task.title}</span>
-          <span> | </span>
-          <span>Due {TimeHelper.toShortHumanDateTimeTz(ux.task.dueAtMoment)}</span>
+          <div><span>{ux.task.title}</span></div>
+          <div><span>|</span></div>
+          <div><span>Due {TimeHelper.toShortHumanDateTimeTz(ux.task.dueAtMoment)}</span></div>
         </StyledHeadingTitle>
```

Comment:
this seems strange to me.  Why wrap each span with a div?  Is it for the `div + div` css rule above?  Seems like we could either set the rules to target span or just remove the inner spans and use div.

Reply:
true. I don't know why I left the span in there. :+1+